docs/pages/Theming/DefaultThemes.md

```diff
@@ -0,0 +1,96 @@
+
+FURN has a default theme built in based on the [web version of the Fluent Design System](https://developer.microsoft.com/en-us/fluentui#/styles/web). This theme can be accessed by calling `createDefaultTheme()`.
+
+We also have a default theme for each platform we support, which makes our components take on more of a native feel, while still allowing the components to fit with other Microsoft experiences.
+
+## Platform Differences
+
+There are slight differences of the default theme between platforms. Each platform has its own default values for typography, spacing, and other styling applied to components.
+
+On Android and Apple platforms, you can access the default themes by calling `createAndroidTheme()`, `createAppleTheme()` respectively. These can be passed into the `ThemeProvider` for use in your app. On Windows, the default theme is provided by `createDefaultTheme()`.
+
+On win32, we allow for better integration with an Office host.
+
+## Integration with Office
+
+On win32, the we allow for integration with Office theming via an `OfficeThemingModule`. Office has a native module which provides information about the queried palette based on the current Office theme. FURN uses the information to populate `theme.colors`, but the palette is available as well under `theme.host.palette`.
+
+You can create a theme using information from Office by calling `createOfficeTheme()`. By default, we use the `WhiteColors` palette. You can also ask for a specific palette's values by passing in a `paletteName` as part of the `ThemeOptions` passed into `createOfficeTheme()`:
+
+```tsx
+const AppWithOfficeTheme = () => {
+  return (
+    // Populate the theme with colors from PaletteName palette in Office.
+    <ThemeProvider theme={createOfficeTheme({ paletteName: 'PaletteName' })}>
+      <AppContent />
+    </ThemeProvider>
+  );
+};
+
+const AppContent = () => {
+  const officeTheme = useTheme();
+  return (
+    <>
+      // bodyText property value of theme is determined by palette
+      <Text color={officeTheme.colors.bodyText}>Hello World!{'\n'}</Text>
+      // Access palette entries directly if required
+      <Text color={officeTheme.host.palette.Text}>Lorem ipsum!</Text>
+    </>
+  );
+};
+```
+
+## Example
+
+Here's how the button component looks by default on every platform:
+
+### Default (Web / Windows)
+
+```tsx
+<ThemeProvider theme={createDefaultTheme()}>
+  <Button content={'Default Button'} />
+</ThemeProvider>
+```
+
+![Image of default button with default theme applied in FURN](./assets/default.PNG)
+
+### Android
+
+```tsx
+<ThemeProvider theme={createAndroidTheme()}>
+  <Button content={'Button'} />
+</ThemeProvider>
+```
+
+![Image of default button with default Android theme applied in FURN](./assets/android.PNG)
+
+### iOS
+
+```tsx
+<ThemeProvider theme={createAppleTheme()}>
+  <Button content={'Button'} />
+</ThemeProvider>
+```
+
+![Image of default button with default Apple theme on iOS applied in FURN](./assets/iOS.PNG)
```

Comment:
Could we make all these files lowercase .png ? I think that's the more standard

Reply:
Uhh sure let me try. Unfortunately the Windows OS doesn't count case changing as a rename so it kinda just keeps the case if I just try to rename

Reply:
Ah. That's probably alright then. I forgot windows case matching is different.

Reply:
Tried something, I think it worked